dts/bindings/sensor/invensense,icm42688.yaml

```diff
@@ -4,111 +4,48 @@
 
-properties:
-  int-gpios:
-    type: phandle-array
-    description: |
-      The INT signal default configuration is active-high. The
-      property value should ensure the flags properly describe the
-      signal that is presented to the driver.
+    Where,
+    ICM42688_DT_GYRO_FS_2000 represents +-2000 deg per sec
+    as the full scale range of the gyroscope.
 
-  accel-pwr-mode:
-    type: int
-    default: 0
-    description: |
-      Specify the default accelerometer power mode.
-      Default is power-up configuration.
-    enum:
-      - 0 # ICM42688_DT_ACCEL_OFF
-      - 2 # ICM42688_DT_ACCEL_LP
-      - 3 # ICM42688_DT_ACCEL_LN
+    ICM42688_DT_ACCEL_FS_16 represents +-16 times g as the maximum
+    acceleration this device can measure.
 
-  accel-odr:
-    type: int
-    default: 6
-    description: |
-      Specify the default accelerometer output data rate expressed in samples per second (Hz).
-      Default is power-up configuration.
-    enum:
-      - 1  # ICM42688_DT_ACCEL_ODR_32000
-      - 2  # ICM42688_DT_ACCEL_ODR_16000
-      - 3  # ICM42688_DT_ACCEL_ODR_8000
-      - 4  # ICM42688_DT_ACCEL_ODR_4000
-      - 5  # ICM42688_DT_ACCEL_ODR_2000
-      - 6  # ICM42688_DT_ACCEL_ODR_1000
-      - 7  # ICM42688_DT_ACCEL_ODR_200
-      - 8  # ICM42688_DT_ACCEL_ODR_100
-      - 9  # ICM42688_DT_ACCEL_ODR_50
-      - 10 # ICM42688_DT_ACCEL_ODR_25
-      - 11 # ICM42688_DT_ACCEL_ODR_12_5
-      - 12 # ICM42688_DT_ACCEL_ODR_6_25
-      - 13 # ICM42688_DT_ACCEL_ODR_3_125
-      - 14 # ICM42688_DT_ACCEL_ODR_1_5625
-      - 15 # ICM42688_DT_ACCEL_ODR_500
+compatible: "invensense,icm42688"
+include: invensense,icm4268x-common.yaml
 
+properties:
   accel-fs:
     type: int
     default: 0
     description: |
       Specify the accelerometer range in g.
       Default is power-up configuration.
     enum:
-      - 0 # ICM42688_DT_ACCEL_FS_16G
-      - 1 # ICM42688_DT_ACCEL_FS_8G
-      - 2 # ICM42688_DT_ACCEL_FS_4G
-      - 3 # ICM42688_DT_ACCEL_FS_2G
-
-  gyro-pwr-mode:
-    type: int
-    default: 0
-    description: |
-      Specify the default gyro power mode.
-      Default is power-up configuration.
-    enum:
-      - 0 # ICM42688_DT_GYRO_OFF
-      - 1 # ICM42688_DT_GYRO_STANDBY
-      - 3 # ICM42688_DT_GYRO_LN
-
-  gyro-odr:
-    type: int
-    default: 6
-    description: |
-      Specify the default gyro output data rate expressed in samples per second (Hz).
-      Default is power-up configuration.
-    enum:
-      - 1  # ICM42688_DT_GYRO_ODR_32000
-      - 2  # ICM42688_DT_GYRO_ODR_16000
-      - 3  # ICM42688_DT_GYRO_ODR_8000
-      - 4  # ICM42688_DT_GYRO_ODR_4000
-      - 5  # ICM42688_DT_GYRO_ODR_2000
-      - 6  # ICM42688_DT_GYRO_ODR_1000
-      - 7  # ICM42688_DT_GYRO_ODR_200
-      - 8  # ICM42688_DT_GYRO_ODR_100
-      - 9  # ICM42688_DT_GYRO_ODR_50
-      - 10 # ICM42688_DT_GYRO_ODR_25
-      - 11 # ICM42688_DT_GYRO_ODR_12_5
-      - 15 # ICM42688_DT_GYRO_ODR_500
+      - 0 # ICM42688_DT_ACCEL_FS_16
```

Comment:
these comments on all the properties are still not actually going into the documentation if they are not part of the description

Reply:
@decsny .. check now